luigi/setup_logging.py

```diff
@@ -40,7 +40,7 @@ def _section(cls, opts):
         """Get logging settings from config file section "logging"."""
         try:
             logging_config = cls.config['logging']
-        except (TypeError, KeyError, NoSectionError):
+        except (TypeError, KeyError, NoSectionError, AttributeError):
```

Comment:
Do we want the attribute error to allow `logging_config = cls.config.options('logging')` like @riga suggested? Or is the best course of action to just return False?

Reply:
We can't configure logging via section in cfg. It have to be False for `LuigiConfigParser`.

Reply:
For section based logging configuring we need dicts in dists, but cfg format doesn't support it as opposite to toml, yaml, json and other good formats.

Reply:
> For section based logging configuring we need dicts in dists, but cfg format doesn't support it as opposite to toml, yaml, json and other good formats.

Is there a difference between Python 2 and 3?

Reply:
It's particular point of cfg format as is. There can be mistake only if cfg-based config has `logging` section, but there is no reason to do it.

Reply:
But don't you introduce a difference between 2 and 3 then? Might not be too important here, but still.

Reply:
Python version doesn't matter. In any python version cfg-based config doesn't contain `logging` section.

Reply:
Then I maybe just don't get why this block exists at all:

https://github.com/spotify/luigi/blob/8278773fe26bbd6cdb9d7b154a1e12fff254c81f/luigi/setup_logging.py#L74-L78

Reply:
It's True for toml-based config with `logging` section for any Python version.

Reply:
Also it will be true for YAML or JSON based configs in future. I hope, we will add it.

Reply:
I would also prefer not to catch `AttributeError` but handle this case explicitly.

Reply:
I've changed it